express-post-json/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+let id = 1;
+const grades = [];
+
+app.use(express.json())
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+  // console.log(req.body)
+  grades.push({...req.body, id})
+  id++
+  res.send('success')
```

Comment:
Here you should be setting a `status` code of `201` and responding with the newly created `grade` object